modules/alloy/src/main/ts/ephox/alloy/api/behaviour/Dragging.ts

```diff
@@ -20,10 +20,10 @@ const Dragging: DraggingBehaviour<any> = Behaviour.createModes({
   extra: {
     // Extra. Does not need component as input.
     snap: (sConfig: SnapConfigSpec<any>): SnapConfig<any> => ({
-      sensor: Fun.constant(sConfig.sensor),
-      range: Fun.constant(sConfig.range),
-      output: Fun.constant(sConfig.output),
-      extra: Fun.constant(Option.from(sConfig.extra))
+      sensor: sConfig.sensor,
```

Comment:
This is an API breaking change, so make sure you add an alloy changelog entry. Either that or consider reverting it 🤷‍♀ 

- [ ] Add changelog with major version bump or revert

Reply:
Changelogged.